src/Reservoir.jl

```diff
@@ -93,11 +93,10 @@ function initreservoir(m::Model, name=nothing)
        	reservoir[:storage0] = zeros(numreservoirs);
      	reservoir[:evaporation] = zeros(numreservoirs, numsteps);
     else
-        rcmax = convert(Vector{Float64}, reservoirdata[:MAXCAP])
-     	rcmax = rcmax*1233.48
+        rcmax = convert(Vector{Float64}, reservoirdata[:MAXCAP])./1000 #data in cubic meters, change to 1000m3
      	reservoir[:storagecapacitymax] = rcmax;
-     	reservoir[:storagecapacitymin] = zeros(numreservoirs);
-        reservoir[:storage0] = rcmax*0.;
+     	reservoir[:storagecapacitymin] = rcmax*0.0#zeros(numreservoirs);
```

Comment:
What do you have against the zeros function?

Reply:
Nothing, but I keep going back and forth between having reservoirs empty or half full at the start  of the optimization/simulation ... Something that should be in the configuration file I suppose. What is your preference? Empty or half full?

Reply:
Changed.